app/_src/production/dp-config/dpp-on-kubernetes.md

```diff
@@ -175,6 +175,57 @@ control capabilities for the sidecar.
 
 ## Lifecycle
 
+{% if_version gte:2.7.x %}
+The lifecycle management has been significantly improved
+with the `SidecarContainers` feature introduced in Kubernetes v1.29.
+
+### Kubernetes sidecar containers
+
+In order to enable the use of this feature with Kuma you should enable the
+`experimental.sidecarContainers`/`KUMA_EXPERIMENTAL_SIDECAR_CONTAINERS` option.
+
+This feature supports adding the injected Kuma container to
+`initContainers` with `restartPolicy: Always`, which marks it as a sidecar
+container. Refer to the
+[Kubernetes docs](https://kubernetes.io/docs/concepts/workloads/pods/sidecar-containers/)
+to learn more about how they work.
+
+In effect, the following lifecycle subsections are irrelevant when using this feature.
+When enabled, the ordering of the sidecar startup and shutdown is enforced by Kubernetes.
+Draining incoming connections gracefully is handled via a `preStop` hook
+on the `kuma-sidecar` container.
+
+However, remember that if the `terminationGracePeriodSeconds` has elapsed, ordering
+and thus correct behavior of the sidecar is no longer guaranteed.
+The grace period should be set long enough that your workload finishes shutdown
+before it elapses.
+
+As always, your application should itself initiate graceful shutdown
+when it receives SIGTERM. In particular, remember that Kubernetes works largely
+asynchronously. So if your application exits "too quickly," it's possible that
```

Comment:
nitpick:

```suggestion
asynchronously. So if your application exits "too quickly", it's possible that
```

Reply:
Welcome to the fun world of competing English style guides https://www.grammarly.com/blog/quotation-marks/ :wink: 
although maybe I just remove the comma